dartagnan/src/main/java/com/dat3m/dartagnan/configuration/Arch.java

```diff
@@ -33,9 +35,21 @@ public static Arch getDefault() {
 	
 	// Used to decide the order shown by the selector in the UI
 	public static Arch[] orderedValues() {
-		Arch[] order = { C11, ARM8, IMM, LKMM, POWER, RISCV, TSO };
+		Arch[] order = { C11, ARM8, IMM, LKMM, POWER, PTX, RISCV, TSO };
 		// Be sure no element is missing
 		assert(Arrays.asList(order).containsAll(Arrays.asList(values())));
 		return order;
 	}
+
+    // used to check if the coherence is not guaranteed to be total in model
+    public static boolean coIsTotal(Arch arch) {
+        Arch[] coNotTotal = {PTX};
+        return !Arrays.asList(coNotTotal).contains(arch);
+    }
+
+    // used to check if supports virtual addressing.
+    public static boolean supVirAdd(Arch arch) {
+        Arch[] supportVirtualAddress = {PTX};
+        return Arrays.asList(supportVirtualAddress).contains(arch);
+    }
```

Comment:
Please use the full name `supportsVirtualAddressing`.

Reply:
Changed